cli_flags.go

```diff
@@ -63,11 +63,10 @@ var (
 		"If zero, monotonic-realtime clock sync will be performed once, " +
 		"on agent startup, but not periodically."
 	sendErrorFramesHelp = "Send error frames (devfiler only, breaks Kibana)"
-	offCPUThresholdHelp = fmt.Sprintf("If set to a value between 1 and %d will enable "+
-		"off-cpu profiling: Every time an off-cpu entry point is hit, a random number between "+
-		"0 and %d is chosen. If the given threshold is greater than this random number, the "+
-		"off-cpu trace is collected and reported.",
-		support.OffCPUThresholdMax-1, support.OffCPUThresholdMax-1)
+	offCPUThresholdHelp = fmt.Sprintf("The per-mille chance for an off-cpu event being reported. "+
```

Comment:
Maybe don't mention reporting here to avoid confusion (off-cpu events can be recorded but then subsequently dropped for other reasons)

```suggestion
	offCPUThresholdHelp = fmt.Sprintf("The per-mille chance for an off-cpu event being recorded. "+
```